setup.py

```diff
@@ -27,19 +28,11 @@
                              'aurora/impden.f90',
                              'aurora/math.f90'])
 
-# use local makefile and avoid numpy's Extension class...
-#cmd = 'make clean; make aurora'
-#result = subprocess.call(cmd, shell=True) 
-
-
 aurora_dir = os.path.dirname(os.path.abspath(__file__))
-with open(os.path.join(aurora_dir, 'aurora', 'version')) as vfile:
-    version = vfile.read().strip()
-
 install_requires = open('requirements.txt').read().split('\n')
     
-setup(name=package_name,
-      version=version,
+config = Configuration(
```

Comment:
@StanczakDominik is this needed because this Configuration class effectively finds the version file automatically, while the `setup` method doesn't?

Reply:
You know, I haven't checked it with just setup and removing the version argument. Sounds like it should work! I'll check it out after some sleep :)